### PR TITLE
Change type/category to kind. Collection add route no longer depends on caxe service.

### PR DESCRIPTION
**Pull request by birdbrained**
Change type/category to kind. Collection add route no longer depends on caxe serivce.

### CODE REVIEW

File: app/controllers/collections/collection/submissions.js [hunk 1]
```diff
@@ -12,17 +13,17 @@ export default Ember.Controller.extend({
             const wf = collection.get('workflow');
             caxe.set('workflow', wf);
             caxe.set('collection', collection);
-            const case_saved_Promise = caxe.save()
-            case_saved_Promise.then((caxe) => {
+            caxe.save().then((caxe) => {
+                debugger;
```
Comment (cameronblandford) [inline app/controllers/collections/collection/submissions.js]: debugger

Comment (birdbrained) [reply app/controllers/collections/collection/submissions.js]: oooooooh

**Comment by birdbrained** [comment]
also display item information in approval wf

File: app/controllers/collections/collection/submissions.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export default Ember.Controller.extend({
 
     caxe: Ember.inject.service(),
 
+
```
Comment (cameronblandford) [inline app/controllers/collections/collection/submissions.js]: remove newline

File: app/controllers/collections/collection/submissions.js [hunk 3]
```diff
@@ -12,17 +13,16 @@ export default Ember.Controller.extend({
             const wf = collection.get('workflow');
             caxe.set('workflow', wf);
             caxe.set('collection', collection);
-            const case_saved_Promise = caxe.save()
-            case_saved_Promise.then((caxe) => {
+            caxe.save().then((caxe) => {
                 this.set('caxe.activeCase', caxe);
-                this.transitionToRoute("collections.collection.add", collection)
+                this.transitionToRoute("collections.collection.add", collection, caxe.id)
```
Comment (cameronblandford) [inline app/controllers/collections/collection/submissions.js]: ;

File: app/controllers/collections/collection/submissions.js [hunk 4]
```diff
@@ -12,17 +13,16 @@ export default Ember.Controller.extend({
...
             })
         },
 
         continueCase(caxe, collection) {
 
             this.set('caxe.activeCase', caxe);
-            this.transitionToRoute('collections.collection.add', collection)
+            this.transitionToRoute('collections.collection.add', collection, caxe.id)
```
Comment (cameronblandford) [inline app/controllers/collections/collection/submissions.js]: ;

**Pull request merged**